cloud/linode/loadbalancers.go

```diff
@@ -643,15 +643,26 @@ func getPortConfig(service *v1.Service, port int) (portConfig, error) {
 			protocol = "tcp"
 		}
 	}
-
 	protocol = strings.ToLower(protocol)
 
+	proxyProtocol := portConfigAnnotation.ProxyProtocol
+	if proxyProtocol == "" {
+		var ok bool
+		for _, ann := range []string{annLinodeDefaultProxyProtocol, annLinodeProxyProtocol} {
+			proxyProtocol, ok = service.Annotations[ann]
+			if ok {
+				break
+			}
+		}
```

Comment:
I just realized that the code as suggested would overwrite deprecated-style annotations with "none" if that was the only one present in the service object. I think you can get around that by keeping the original code, and doing something like
```
proxyProtocol = linodego.ProxyProtocolNone
```
before the loop.

Reply:
This is a good call. But the code above would exit immediately even if the deprecated annotation was configured. I'm gonna default to none below the loop if `!ok`, because that would mean that neither of the annotations are configured.